src/System.Management.Automation/engine/Attributes.cs

```diff
@@ -1014,6 +1014,11 @@ public sealed class ValidateRangeAttribute : ValidateEnumeratedArgumentsAttribut
         /// </summary>
         private Type _promotedType;
 
+        /// <summary>
+        /// Gets the name of the predefined range.
+        /// </summary>
+        public ValidateRangeKind? RangeKind { get; }
```

Comment:
```suggestion
        public ValidateRangeKind? RangeKind { get => _rangeKind; }
```

May as well keep the properties together, no? 😄

Reply:
Are those really done like that? I didn't look at the whole file... Guess I might do a style pass over this when this is done 😂

Reply:
Please use "internal".

Reply:
How does this relate to this fix?

Reply:
My understanding is that we can fix the issue without changing APIs and we should do so.
New public API is another issue and should be discussed and approved separately.